src/tools/compiletest/src/runtest.rs

```diff
@@ -224,6 +224,19 @@ pub fn run(config: Config, testpaths: &TestPaths, revision: Option<&str>) {
 pub fn compute_stamp_hash(config: &Config) -> String {
     let mut hash = DefaultHasher::new();
     config.stage_id.hash(&mut hash);
+    match config.mode {
+        DebugInfoGdb => match config.gdb {
+            None => env::var_os("PATH").hash(&mut hash),
+            Some(ref s) if s.is_empty() => env::var_os("PATH").hash(&mut hash),
+            Some(ref s) => s.hash(&mut hash),
+        },
+        DebugInfoLldb => {
+            env::var_os("PATH").hash(&mut hash);
+            env::var_os("PYTHONPATH").hash(&mut hash);
+        },
```

Comment:
Hm, why do these need to be two separate paths? I feel like we can hash equally independent of whether we have gdb configured; if we rerun the path uselessly that's more or less fine I think.

Reply:
It was just more precise; and also gdb can be configured with an explicit path in `config.toml`.  However, if you want both paths to be the same, that's easy enough to do.